tools/allowed-large-files

```diff
@@ -40,6 +40,8 @@ roxctl/helm/derivelocalvalues/testdata/k8s-resources/*.yaml
 scale/image/fixtures/scrape.json
 scanner/e2etests/testdata/image_tests.json
 scanner/e2etests/testdata/image_tests.orig.json
+scanner/indexer/mappingfiles/container-name-repos-map.json
+scanner/indexer/mappingfiles/repository-to-cpe.json
```

Comment:
Can we create an issue to add these files to the container image instead and pull them during the image build?

Reply:
> add these files to the container image instead and pull them during the image build

I am not sure how to do that. Is there any examples to look at?

Reply:
Just create the JIRA ticket so we can do it as soon as we can.

Reply:
https://issues.redhat.com/browse/ROX-21690

cc: @jvdm  @jvdm

Reply:
https://github.com/stackrox/stackrox/pull/9362

Reply:
Please coordinate this change against #9362. One thing to think about is that by having the mappings in the container image, we cannot run the scanner locally anymore unless we can either ignore the files if they are not found or make their path configurable in the config.